tools/build/package

```diff
@@ -170,6 +177,7 @@ def _main(argv):
     print("Building release for platform: " + platform)
 
     manifest = _read_manifest(root_dir, platform, versions)
+    print(manifest)
```

Comment:
Could this be a debug printout that was forgotten?

Reply:
Yes, thanks!